src/overlays/actors/ovl_En_Ru/z_en_ru.c

```diff
@@ -76,34 +86,292 @@ static DamageTable D_80A39488 = {
     /* Powder Keg     */ DMG_ENTRY(0, 0x0),
 };
 
-#endif
+static AnimationInfoS sAnimations[] = {
+    { &gAdultRutoIdleAnim, 1.0f, 0, -1, ANIMMODE_LOOP, 0 },
+    { &gAdultRutoIdleAnim, 1.0f, 0, -1, ANIMMODE_LOOP, -4 },
+    { &gAdultRutoRaisingArmsUpAnim, 1.0f, 0, -1, ANIMMODE_LOOP, -4 },
+    { &gAdultRutoCrossingArmsAnim, 1.0f, 0, -1, ANIMMODE_LOOP, -4 },
+    { &gAdultRutoLookingDownLeftAnim, 1.0f, 0, -1, ANIMMODE_LOOP, -4 },
+    { &gAdultRutoIdleHandsOnHipsAnim, 1.0f, 0, -1, ANIMMODE_LOOP, -4 },
+    { &gAdultRutoHeadTurnDownLeftAnim, 1.0f, 0, -1, ANIMMODE_LOOP, -4 },
+    { &gAdultRutoSwimmingUpAnim, 1.0f, 0, -1, ANIMMODE_LOOP, -4 },
+};
+
+// in PostLimbdraw, conversts limbIndex to bodyPartsPos index
+s8 sBodyPartPosIndexs[] = { -1, -1, 0x0C, 0x0D, 0x0E, -1, 0x09, 0x0A, 0x0B, -1,  0x00, 0x06,
```

Comment:
I think the plural of `index` is `indices`, also this name is kinda generic so it may be a good idea to use `static` to avoid crashes